src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1914 @@
+            hs.schedule_hsi_event(
+                hsi_event=HSI_CervicalCancer_PalliativeCare(
+                    module=self.module,
+                    person_id=person_id
+                ),
+                topen=self.sim.date + DateOffset(months=1),
+                tclose=None,
+                priority=0
+            )
+
+# ---------------------------------------------------------------------------------------------------------
+#   LOGGING EVENTS
+# ---------------------------------------------------------------------------------------------------------
+
+class CervicalCancerLoggingEvent(RegularEvent, PopulationScopeEventMixin):
+    """The only logging event for this module"""
+
+    # the use of groupby might be more efficient in computing the statistics below;
+
+    def __init__(self, module):
+        """schedule logging to repeat every 1 month
+        """
+        self.repeat = 30
+        super().__init__(module, frequency=DateOffset(days=self.repeat))
+
+    def apply(self, population):
+        """Compute statistics regarding the current status of persons and output to the logger
+        """
+        df = population.props
+        p = self.sim.modules['CervicalCancer'].parameters
+
+
+        # CURRENT STATUS COUNTS
+        # Create dictionary for each subset, adding prefix to key name, and adding to make a flat dict for logging.
+        out = {}
+
+        self.sim.date - pd.DateOffset(days=29)
+
+        # Current counts, total
+        out.update({
+            f'total_{k}': v for k, v in df.loc[df.is_alive & (df['sex'] == 'F') &
+                                               (df['age_years'] > p['min_age_hpv'])].ce_hpv_cc_status.value_counts().items()})
+
+        # Current counts, total hiv negative
+        out.update({
+            f'total_hivneg_{k}': v for k, v in df.loc[df.is_alive & (df['sex'] == 'F') &
+                                               (df['age_years'] > p['min_age_hpv']) & (~df['hv_inf'])].ce_hpv_cc_status.value_counts().items()})
+
+        # Current counts, total hiv positive
+        out.update({
+            f'total_hivpos_{k}': v for k, v in df.loc[df.is_alive & (df['sex'] == 'F') &
+                                               (df['age_years'] > p['min_age_hpv']) & (df['hv_inf'])].ce_hpv_cc_status.value_counts().items()})
+
+        out.update({
+            'total_males': len(df[df.is_alive & (df['sex'] == 'M')])})
+        out.update({
+            'total_dead': len(df[df['is_alive'].eq(False)])})
+        out.update({
+            'total_overall': len(df)})
+
+        # Get the day of the year
+        day_of_year = self.sim.date.timetuple().tm_yday
+
+        # Calculate the decimal year
+        decimal_year = self.sim.date.year + (day_of_year - 1) / 365.25
+        rounded_decimal_year = round(decimal_year, 2)
+
+        df['rounded_decimal_year'] = rounded_decimal_year
+
+        date_1_year_ago = self.sim.date - pd.DateOffset(days=365)
```

Comment:
```suggestion
        date_1_year_ago = self.sim.date - pd.DateOffset(years=1)
```
I think years/months is better since it takes into account leap years?